sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -668,11 +668,11 @@ export type MetricAnomalyFeedback = {
   /**
    * the start timestamp of feedback timerange
    */
-  startTime: Date;
+  startTime?: Date;
```

Comment:
are `startTime` and `endTime` optional in the portal for Anomaly Feedback?  I know service team unified them but we can still make them required to better reflect what's required.

Reply:
Yes they did it before but fxed it later. Thanks for pointing it, i removed the optional